ppa/archive/models.py

```diff
@@ -97,19 +120,13 @@ def populate_from_bibdata(self, bibdata):
         # - last update, rights code / rights string, item url
         # (maybe solr only?)
 
-    def index(self, commit=False):
+    def index(self, params=None):
```

Comment:
Seems potentially dangerous to allow open-ended params that we pass through directly to Solr; we may want to revisit at some point, but we can wait and see how this works for us.